projects/max14906/src/examples/basic/basic_example.c

```diff
@@ -173,16 +155,18 @@ int basic_example_main()
 	/** Setting the on state for channel 0, and then verifying it. */
 	if (max14906_desc->en_gpio) {
 		ret = max14906_ch_set(max14906_desc, 0, 1);
-		if(ret)
+		if (ret)
```

Comment:
unrelated changes, please create a new commit for this

Reply:
Adding new commit.